index.html

```diff
@@ -53,6 +63,7 @@ <h2>My Recent Works</h2>
         <div class="work-container">
```

Comment:
It's required to stick with the Figma template, so please make sure to replace the light gray background with the white background color.

![image](https://user-images.githubusercontent.com/34779928/193095911-138429dc-9d29-4dbf-a1ea-21420e2e9c0a.png)